androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/home/HomeScreen.kt

```diff
@@ -98,15 +100,17 @@ fun HomeContent(
     isLoading: Boolean,
     surveyUiModels: List<SurveyUiModel>,
     onUserAvatarClick: () -> Unit,
+    onSurveyClick: (surveyUiModel: SurveyUiModel) -> Unit,
     onRefresh: () -> Unit,
     onPageChange: (page: Int) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    var isFirstLaunch by remember { mutableStateOf(true) }
```

Comment:
This logic is not working. I am seeing the pull to refresh indicator at the first launch. Can you please check?

Reply:
Resolved in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/118/commits/a3308a7226d9aad51af582cdbff6ff030b18005a